server/container/guice/guice-common/src/main/java/org/apache/james/modules/protocols/JMAPServerModule.java

```diff
@@ -19,20 +19,20 @@
 
 package org.apache.james.modules.protocols;
 
+import java.security.Security;
+
 import org.apache.james.jmap.JMAPModule;
 import org.apache.james.jmap.JMAPServer;
 import org.apache.james.jmap.crypto.JamesSignatureHandler;
 import org.apache.james.mailbox.store.mail.model.MailboxId;
 import org.apache.james.utils.ConfigurationPerformer;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
 
 import com.google.inject.AbstractModule;
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
 import com.google.inject.TypeLiteral;
 import com.google.inject.multibindings.Multibinder;
-import org.bouncycastle.jce.provider.BouncyCastleProvider;
-
-import java.security.Security;
```

Comment:
GitHub shows the difference only when clicking on `View full changes` -> import ordering